File: app/src/main/java/com/devjakkifx/frasesdodia/MainActivity.java
package com.devjakkifx.frasesdodia;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    // Atributos
    String[] frases = {
            "Frase 01",
            "Frase 02",
            "Frase 03",
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void gerarFrase(View view) {
        TextView texto = findViewById(R.id.txtFrase);
        int numRamdom = new Random().nextInt(3);
        String frase = frases[numRamdom];
        texto.setText( frase );
    }

    public void exibirTodas(View view) {
        TextView texto = findViewById(R.id.txtFrase);

        String txtFrase = "";
        for (String frase : frases) {
            txtFrase += frase;
        }

        texto.setText(txtFrase);
    }
}

File: app/src/main/java/com/devjakkifx/projeto1/MainActivity.kt
package com.devjakkifx.projeto1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import java.util.Random

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sortear(view: View) {
        val txtResult = findViewById<TextView>(R.id.text_result); // Seleciona um elemento da interface pelo ID
        val number = Random().nextInt(10); // Gera um número inteiro aleatório, no caso, de 0 a 10
        txtResult.setText("Número gerado: $number");
    }
}